### PR TITLE
Adding placeholders for custom NSP vues

### PR DESCRIPTION
**Pull request by davidjumani**
### Description

Adds placeholder text for NSPs with custom vues

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

### Screenshots (if appropriate):

TODO

### CODE REVIEW

**Comment by rhtyd** [comment]
ping @davidjumani is this ready?

**Comment by davidjumani** [comment]
@blueorangutan ui

File: ui/src/views/infra/network/providers/AddSrxFirewall.vue [hunk 1]
```diff
@@ -25,8 +25,9 @@
         <a-col :md="24" :lg="24">
           <a-form-item :label="$t('label.ip')">
             <a-input
+              :placeholder="apiParams.url.description"
               autoFocus
-              v-decorator="['ip', {
+              v-decorator="['url', {
```
Comment (utchoang) [inline ui/src/views/infra/network/providers/AddSrxFirewall.vue:30]: @davidjumani If you change it like this, the current handleSubmit() function is still using `values.ip`?

**Review by utchoang — CHANGES_REQUESTED** [review]

**Comment by sureshanaparti** [comment]
Hi @davidjumani, Can you address the outstanding comments.

**Comment by sureshanaparti** [comment]
ping @davidjumani please address the outstanding comments. thanks.

File: ui/src/views/infra/network/providers/AddSrxFirewall.vue (hunk 1 above)
Comment (davidjumani) [reply ui/src/views/infra/network/providers/AddSrxFirewall.vue:30]: Thanks for the catch @utchoang. Pushing a fix

**Comment by davidjumani** [comment]
@blueorangutan ui

**Review by utchoang — APPROVED** [review]
LGTM! Thanks @davidjumani

**Review by shwstppr — APPROVED** [review]

**Comment by sureshanaparti** [comment]
Verified the placeholder descriptions from the API params.

@davidjumani Can you update the description for the below param in addNetscalerLoadBalancer API.

https://github.com/apache/cloudstack/blob/b9905c7e377bc9c5f5129e7d43be40183a06c052/plugins/network-elements/netscaler/src/main/java/com/cloud/api/commands/AddNetscalerLoadBalancerCmd.java#L86

**Comment by davidjumani** [comment]
@blueorangutan package

**Review by sureshanaparti — APPROVED** [review]
LGTM

**Comment by sureshanaparti** [comment]
@blueorangutan package

**Pull request merged**